public/stylesheets/register-new-user.css

```diff
@@ -0,0 +1,30 @@
+body {
```

Comment:
Created a global.css file that modifies the body. Right now it just includes font and padding at the top, but we can add more.